package.json

```diff
@@ -325,8 +328,7 @@
     "schedule": "0.5.0",
     "ts-migrate": "0.1.26",
     "typescript": "4.4.4",
-    "typescript-coverage-report": "0.6.1",
-    "husky": "8.0.1"
+    "typescript-coverage-report": "0.6.1"
   },
```

Comment:
can you do something with those changes here?

Reply:
sure thing, I have no idea why this mixed up